apps/designer/app/canvas/shared/use-drag-drop.ts

```diff
@@ -136,6 +136,18 @@ export const useDragAndDrop = () => {
         },
       });
     },
+
+    getValidChildren: (parent) => {
+      if (!(parent instanceof HTMLBodyElement)) {
```

Comment:
why is body children not valid? is it possible to be more generic? e.g. not have assumptions about body, just have a check if an element is a ws instance?

Reply:
I'll use `getInstanceIdFromElement`. It would work well after we switch to a unique `data` attribute. And currently should be fine too as the `<script>` and `<link>` elements that are inserted don't have `id`s.